.travis.yml

```diff
@@ -62,7 +62,8 @@ matrix:
       - TYPE="RUBY"
 before_install:
   - |
-      if ! git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(.md)|(.png)|(.pdf)|(.html)|^(LICENSE)|^(docs)'
+      echo "Checking if a CI run is needed post commit: ${TRAVIS_COMMIT_RANGE}"
+      if ! git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(\.md$)|(\.png$)|(\.pdf$)|(\.html$)|^(LICENSE)|^(docs)|(\.yml$)'
```

Comment:
my guess is if you update .yml, you will still need to run CI to make sure travis.yml is not broken? or will it be picked up by travis for any travis.yml change